packages/expo-document-picker/ios/DocumentPickerResponse.swift

```diff
@@ -1,5 +1,7 @@
 import ExpoModulesCore
 
+// Disable swiftlint for redundant_optional_initialization. Optional initialization with nil is necessary for the Record constructors to work.
+// swiftlint:disable  redundant_optional_initialization
```

Comment:
```suggestion
// Disable swiftlint for redundant_optional_initialization. Optional initialization with nil is necessary for the Record constructors to work.
// swiftlint:disable redundant_optional_initialization
```

Reply:
The package you referenced does not use nil.
But [expo-print has it](https://github.com/expo/expo/blob/54d745bed821a8054a52237739a7b4571448bf89/packages/expo-print/ios/PrintResponse.swift)